packages/astro/test/test-utils.js

```diff
@@ -69,7 +69,15 @@ export async function loadFixture(inlineConfig) {
 	}
 	// Load the config.
 	let config = await loadConfig({ cwd: fileURLToPath(cwd) });
-	config = merge(config, { ...inlineConfig, projectRoot: cwd });
+	config = merge(config, { ...inlineConfig, root: cwd });
+
+	// Note: the inline config doesn't run through config validation where these normalizations usually occur
```

Comment:
Ah ha! Glad it was easy to re-create validation here. Still confused why we couldn't import core's `validateConfig` for this... but this seems like a fine stand-in.